### PR TITLE
Created models and migration.

### PR DESCRIPTION
**Pull request by SadouskiYury**

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -178,7 +178,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 6..128
```
Comment (markevich) [inline config/initializers/devise.rb:181]: Верни блэт

**Review by markevich — APPROVED** [review]

File: config/initializers/devise.rb (hunk 1 above)
Comment (SadouskiYury) [reply config/initializers/devise.rb]: doone

File: app/controllers/admin_masters_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  def edit; end
+
+  # POST /admin_masters or /admin_masters.json
+  def create
+    @admin_master = AdminMaster.new(admin_master_params)
+
+    respond_to do |format|
+      if @admin_master.save
+        format.html { redirect_to admin_master_url(@admin_master), notice: "Admin master was successfully created." }
+        format.json { render :show, status: :created, location: @admin_master }
```
Comment (markevich) [inline app/controllers/admin_masters_controller.rb:29]: do you need a json response overall? You can remove it in next PR if you are not planning to implement any API

Comment (SadouskiYury) [reply app/controllers/admin_masters_controller.rb:29]: wil be update in th next mr

File: app/models/master.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  has_many :articles, dependent: :nullify
+  has_many :reviews, dependent: :nullify
+  has_many :contacts, dependent: :destroy
+  has_many :services, dependent: :destroy
+  has_many :discounts, dependent: :destroy
+
+  has_many :admin_masters, dependent: :destroy
+  has_many  :admins, through: :admin_masters
+  # rubocop:disable Rails/UniqueValidationWithoutIndex
+  validates :name, presence: true, uniqueness: { case_sensitive: true }
```
[the remark this answers is not in the document]
Comment (SadouskiYury) [reply app/models/master.rb]: fixed

**Pull request merged**